gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -314,6 +325,11 @@ private static GoogleCloudStorageItemInfo getObjectMetadata(
     } catch (InterruptedException e) {
       Thread.currentThread().interrupt();
       throw new IOException("Thread interrupt received.", e);
+    } finally {
+      long endTime = System.currentTimeMillis();
+      logger.atFinest().log(
+          "Spent %d milliseconds to fetch metadata for resource %s",
```

Comment:
For consistency and ease of grepping may I suggest the below message format
```
prefetched footer, resource:%s, time:%d, size:%d
fetched metadata, resource:%s, time:%d
read data, resource:%s, time:%d, requested:%d, offset:%d, remaining:%d
```

Reply:
Done